package.json

```diff
@@ -22,11 +22,11 @@
     "tinymce": "^6.2.0"
   },
   "devDependencies": {
-    "babel-core": "^6.26.3",
     "@silverstripe/eslint-config": "^1.3.0",
-    "@silverstripe/webpack-config": "^3.0.0-alpha4",
+    "@silverstripe/webpack-config": "^3",
     "@testing-library/dom": "^10.4.0",
     "@testing-library/react": "^16.1.0",
+    "babel-core": "^6.26.3",
```

Comment:
`babel-core` moving must have been something `yarn` wanted to do. All I did was `yarn upgrade @silverstripe/webpack-config@^3 && yarn build`

Reply:
Yeah it's just alpha sorted, all good